NetCoreRestApi/DataLayer/Converters/CategoryEntityConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DataLayer.Models;
+using Common.Converter;
+using DataLayer.Entities;
+
+namespace DataLayer.Converters
+{
+    public class CategoryEntityConverter : IConverter<CategoryEntity, CategoryModel>
+    {
+        public CategoryModel ConvertFrom(CategoryEntity categoryEntity)
```

Comment:
Again we should understand that we convert to entity from model so the method should accept model and return entity

Reply:
done